src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -7,6 +7,9 @@ public class Manufacturer {
     private String name;
     private String country;
 
+    public Manufacturer() {
+    }
+
     public Manufacturer(Long id, String name, String country) {
```

Comment:
do you use this constructor somewhere?

Reply:
Yes, when i parse manufacturer drom resultset and in main